config/locales/diaspora/en.yml

```diff
@@ -1039,6 +1039,13 @@ en:
       reshare_by: "Reshare by %{author}"
 
   report:
+    modal:
+      title: "You are about to send an email to all %{pod_name} moderators"
```

Comment:
> If you want a place-holder, I suggest 'Enter your report here'. Or, you could enter "Please state how the content you are reporting violates our terms of use." from reason_label as a place-holder.

Exactly, a request to act, not just an empty space. 
Let us keep in mind (not for this PR) that other networks offer their reportes a list of (most used) topics. If nothing fit, user can give a text. 

For an analysis by moderators or admins, its good to keep this texts / categories short. But that's another Issue not to solve here.

Reply:
What about "This content violates the terms because..."? Placeholder are used to give an example of bad content, but I don't want to specify one as the terms are specific to the pod so I can't know what is forbidden, apart probably child porn but I don't want to mention this.